R/logistic_regression.R

```diff
@@ -373,7 +373,7 @@ h_simple_term_labels <- function(terms,
   assertthat::assert_that(
     is.table(table)
   )
-  assert_character_or_factor(terms)
+  checkmate::assertMultiClass(terms, classes = c("factor", "character"))
```

Comment:
just to be consistent by using snake_case only
```suggestion
  checkmate::assert_multi_class(terms, classes = c("factor", "character"))
```

Reply:
done in the next commit! thanks.